api-samples/webRequest/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "webRequest Demo",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
```

```suggestion
    "service_worker": "service-worker.js"
```
We would probably start using `service-worker.js` in our examples